### PR TITLE
sample.kind = 'Rounding' is optional

### PR DESCRIPTION
**Pull request by Kucharssim**
See https://github.com/jasp-stats/jaspBase/pull/101

### CODE REVIEW

File: R/run.R [hunk 1]
```diff
@@ -67,6 +67,10 @@ runAnalysis <- function(name, dataset, options, view = TRUE, quiet = FALSE, make
       stop("Please supply an analysis name")
   }
 
+  if (isTRUE(.Options[["jaspLegacyRngKind"]]) || is.null(.Options[["jaspLegacyRngKind"]])) {
+    message("Legacy RNG is used by default. To use your `RNGkind()` settings instead, use `options(jaspLegacyRngKind = FALSE)`.")
```
Comment (vandenman) [inline R/run.R:71]: `warning`? Also, this probably needs `domain = NA`.

**Review by vandenman — APPROVED** [review]
looks good, let's hope people will also regenerate the tests

**Comment by Kucharssim** [comment]
> let's hope people will also regenerate the tests

We'll probably need to make an announcement for this, and at some point switch the default to the R's default so that the switch becomes opt-out rather than opt-in...

File: R/run.R (hunk 1 above)
Comment (shun2wang) [reply R/run.R:71]: you will never use it after this [PR.](https://github.com/jasp-stats/jasp-actions/pull/16#issuecomment-1313539763) :-)

**Pull request merged**